components/molecules/chanel/navigation/base.vue

```diff
@@ -1,18 +1,10 @@
 <template>
-  <NuxtLink class="group">
-    <AtomsCenteredSquare
-      class="rounded-[50%] bg-gray-800 text-gray-300 transition-all ease-in-out active:translate-y-0.5 group-hover:rounded-2xl group-hover:bg-blue-600 aria-selected:rounded-2xl aria-selected:bg-blue-600"
-      :aria-selected="selected"
-    >
+  <NuxtLink
+    class="group overflow-hidden rounded-[50%] bg-gray-800 transition-all ease-in-out hover:rounded-2xl hover:bg-blue-600 active:translate-y-0.5"
+    active-class="!rounded-2xl !bg-blue-600"
```

Comment:
why do we need '!' before the class name here?)

Reply:
good question. it does not work without it! :D

Reply:
👍🏻